### DISCUSSION TITLE
Wazuh dashboard users activity logs

### DISCUSSION THREAD

**Discussion by EToLLa** [wazuh/wazuh#16225] [Q&A] [answered]
Hello.
I'm trying to recieve logs from user activity in wazuh dashboard, such as sign in/sign out, changing alerts, performing searches and so on.
I turned on audit logs in opensearch part of wazuh dashboard, but still can't see any logs. I can't find a solution on the internet. (
Can anyone help with answer ? Is it possible to logging user's actions in wazuh dashboard, and how to turn on it ?

**Comment by carlosdams** [comment, marked as answer]
Hi @EToLLa ,
To enable audit logs of OpenSearch you must have `plugins.security.audit.type: internal_opensearch` in the file `/etc/wazuh-indexer/opensearch.yml`

Then check the option "Enable audit logging" under the hamburger icon > Security > Audit logs. From this page you can edit the settings for audit logs too, if you want the Authenticated events to appear you will have to make sure this is not disabled in General Settings.
For a full list of the type of events you can check the official documentation for Audit logs:
https://opensearch.org/docs/1.2/security-plugin/audit-logs/index/#tracked-events

The last step to visualize these events is to make sure the index pattern for `security-auditlog*` is created, expand the hamburger icon > Stack Management > Index patterns, in case there is no security-auditlog* already created then you have to create it.
![image](https://user-images.githubusercontent.com/35348824/219394495-8330cda7-63c5-47ff-8d0d-12088d9d941c.png)

You will be able to visualize the events from OpenSearch Dashboards > Discover, make sure you select the `security-auditlog*` index pattern
![image](https://user-images.githubusercontent.com/35348824/219395078-ee6850e1-25af-496f-aa96-e92e23692d33.png)

I hope you find this information useful, please let me know

**Reply by EToLLa** [reply]
Hello, @carlosdams.
Thank you very much! It worked perfectly for me!

**Reply by PotassiumPermanganate** [reply]
@carlosdams Great Answer! Is there any why I can generate email alerts using the above solution?

**Reply by kreshanths-mafiree** [reply]
Thank you so much, @carlosdams
